purchases/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -335,6 +336,60 @@ class BackendTest {
         }
     }
 
+    @Test
+    fun `given getCustomerInfo call on foreground, then one in background, only one request without delay is triggered`() {
```

Comment:
we have to do this because it'd be tricky to cancel the existing request for the background, right?